packages/forms-web-app/src/routes/file-based-router/comment-planning-appeal/add-comments/controller.js

```diff
@@ -1,10 +1,15 @@
+/**
+ * @typedef {import('../check-answers/controller')} InterestedParty
+ */
+
 /** @type {import('express').RequestHandler} */
 const addCommentsGet = (req, res) => {
 	if (!req.session.interestedParty?.appealNumber) {
 		return res.redirect(`enter-appeal-reference`);
 	}
 
-	const interestedParty = req.session.interestedParty || {};
+	/** @type {InterestedParty} */
+	const interestedParty = req.session.interestedParty;
```

Comment:
if you're still going to be working on this can you move lookups/setting of data for IP session into 1 service file
makes it a lot easier to find/change what's on the session object at a later date